src/index.html

```diff
@@ -158,6 +158,15 @@ <h3>Find content you can share, use, and remix.</h3>
               placeholder="Enter your search query" />
             <button class="search-button">Search</button>
           </div>
+          <div class="body-search-description" role="contentinfo">
+            <p>
```

Comment:
Let's reword this to: 

The Search Portal helps you search CC licensed works across multiple platforms.

Reply:
Let's also remove the "learn more" sentence at the end for now.